vignettes/DeeDeeExperiment_manual.Rmd

```diff
@@ -79,9 +79,9 @@ using the class.
 
 ## Anatomy of DeeDeeExperiment
 
-The `DeeDeeExperiment` class inherits from the core Bioconductor `SummarizedExperiment`
+The `DeeDeeExperiment` class inherits from the core Bioconductor `SingleCellExperiment`
 object. This means it inherits all the slots, methods, and compatibility with
-tools built for `SummarizedExperiment`, while introducing additional components
+tools built for `SingleCellExperiment`, while introducing additional components
 to streamline downstream analysis.
 
 ![](DeeDeeExperiment_Anatomy.png){width=100%}
```

Comment:
Especially here, about the anatomy: Shall we consider adding somewhere also some kind of representation for the redDims? Since this one of the biggest bits of motivating our choice. We could basically simply update the png, I hope it is a feasible edit?